src/main/webapp/definitions/alfresco-core.yaml

```diff
@@ -9232,3 +9235,92 @@ definitions:
       id:
         type: string
         description: The unique identifier of the action pending execution
+  Definition:
+    required:
+      - typeId
+    properties:
+      typeId:
+        description: the id of the type
+        type: string
+      title:
+        description: the human-readable class title
+        type: string
+      parentName:
```

Comment:
We may want to call this one parentTypeId but lets see with some real data first.